### PR TITLE
CLOUD-81127 Simplify LDAP Setup (Ranger, Knox, Zeppelin, Hive)

### PR DESCRIPTION
**Pull request by shwethags**

### CODE REVIEW

File: core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp [hunk 1]
```diff
@@ -128,10 +143,19 @@
...
             "ranger.jpa.jdbc.url": "jdbc:postgresql://{{ RANGER_DB_HOST }}/{{ RANGER_DB_NAME }}",
             "ranger.audit.source.type": "solr",
-            "ranger.authentication.method": "ACTIVE_DIRECTORY",
-            "ranger.ldap.ad.domain": "{{ LDAP_DOMAIN }}",
-            "ranger.ldap.ad.base.dn": "{{ LDAP_SYNC_SEARCH_BASE }}",
-            "ranger.ldap.ad.user.searchfilter": "({{ LDAP_NAME_ATTRIBUTE }}={0})"
+            "ranger.authentication.method": "{{ LDAP_METHOD }}",
+            "ranger.ldap.ad.url": "{{ LDAP_URL }}",
+            "ranger.ldap.ad.base.dn": "{{ LDAP_USER_SEARCH_BASE }}",
+            "ranger.ldap.ad.user.searchfilter": "{{ LDAP_USER_SEARCH_FILTER }}",
```
Comment (akanto) [inline core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp]: We don't need search filter, we can use:  "ranger.ldap.ad.user.searchfilter": "({{ LDAP_NAME_ATTRIBUTE }}={0})"

File: core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp [hunk 2]
```diff
@@ -117,9 +129,12 @@
           "ranger.usersync.ldap.url": "{{ LDAP_URL }}",
           "ranger.usersync.ldap.binddn": "{{ LDAP_BIND_DN }}",
           "ranger.usersync.ldap.ldapbindpassword": "{{ LDAP_BIND_PASSWORD }}",
-          "ranger.usersync.ldap.user.nameattribute": "{{ LDAP_NAME_ATTRIBUTE }}",
-          "ranger.usersync.ldap.user.searchbase": "{{ LDAP_SYNC_SEARCH_BASE }}",
-          "ranger.usersync.group.searchenabled": "false"
+          "ranger.usersync.ldap.user.nameattribute": "{{ LDAP_USERNAME_ATTRIBUTE }}",
+          "ranger.usersync.ldap.user.searchbase": "{{ LDAP_USER_SEARCH_BASE }}",
+          "ranger.usersync.ldap.user.searchfilter": "{{ LDAP_USER_SEARCH_FILTER }}",
```
Comment (akanto) [inline core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp]: We don't need search filter, we can use: "ranger.ldap.ad.user.searchfilter": "({{ LDAP_NAME_ATTRIBUTE }}={0})"

File: core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp [hunk 3]
```diff
@@ -128,10 +143,19 @@
...
+            "ranger.ldap.ad.base.dn": "{{ LDAP_USER_SEARCH_BASE }}",
+            "ranger.ldap.ad.user.searchfilter": "{{ LDAP_USER_SEARCH_FILTER }}",
+            "ranger.ldap.ad.bind.dn": "{{ LDAP_BIND_DN }}",
+            "ranger.ldap.ad.bind.password": "{{ LDAP_BIND_PASSWORD }}",
+            "ranger.ldap.url": "{{ LDAP_URL }}",
+            "ranger.ldap.base.dn": "{{ LDAP_USER_SEARCH_BASE }}",
+            "ranger.ldap.bind.dn": "{{ LDAP_BIND_DN }}",
+            "ranger.ldap.bind.password": "{{ LDAP_BIND_PASSWORD }}",
+            "ranger.ldap.group.searchbase": "{{ LDAP_GROUP_SEARCH_BASE }}",
+            "ranger.ldap.user.searchfilter": "{{ LDAP_USER_SEARCH_FILTER }}",
```
Comment (akanto) [inline core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp]: We don't need search filter, we can use: "ranger.ldap.ad.user.searchfilter": "({{ LDAP_NAME_ATTRIBUTE }}={0})"

File: core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp (hunk 2 above)
Comment (shwethags) [inline core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp]: Its empty by default, and that works. So, its fine if we don't set

**Comment by akanto** [comment]
Please send the pull request to rc-1.15 branch

**Comment by shwethags** [comment]
This pull request is for rc-1.15 branch

File: core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp (hunk 1 above)
Comment (doktoric) [inline core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp]: @shwethags can you please also modify this ?

Comment (shwethags) [inline core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp]: removed already

**Pull request closed without merging**